std/net/curl.d

```diff
@@ -1,6 +1,10 @@
 // Written in the D programming language.
 
 /**
+$(MESSAGE_BOX gray, $(LINK2 https://github.com/ikod/dlang-requests, `requests`) provides
+a simple, high-level API with a native D implementation. Moreover, `requests`
+can be used with $(LINK2 http://vibed.org/, Vibe.d) for asynchronous requests.)
```

Comment:
This won't happen:

http://docs.python-requests.org/en/master/dev/philosophy/#standard-library

> Requests has no active plans to be included in the standard library. This decision has been discussed at length with Guido as well as numerous core developers.
> Essentially, the standard library is where a library goes to die. It is appropriate for a module to be included when active development is no longer necessary.

(philosophy is similar)

See also: https://github.com/ikod/dlang-requests/issues/10